Library/Homebrew/hardware.rb

```diff
@@ -96,50 +98,59 @@ def family
         :dunno
       end
 
+      sig { returns(T.nilable(Integer)) }
```

Comment:
```suggestion
      sig { returns(Integer) }
```

should hopefully work